chart/tests/test_airflow_common.py

```diff
@@ -268,3 +268,46 @@ def test_have_all_config_mounts_on_init_containers(self):
         }
         for doc in docs:
             assert expected_mount in jmespath.search("spec.template.spec.initContainers[0].volumeMounts", doc)
+
+    def test_priority_class_name(self):
+        docs = render_chart(
+            values={
+                "flower": {"priorityClassName": "low-priority-flower"},
+                "pgbouncer": {"priorityClassName": "low-priority-pgbouncer"},
+                "scheduler": {"priorityClassName": "low-priority-scheduler"},
+                "statsd": {"priorityClassName": "low-priority-statsd"},
+                "triggerer": {"priorityClassName": "low-priority-triggerer"},
+                "webserver": {"priorityClassName": "low-priority-webserver"},
+                "workers": {"priorityClassName": "low-priority-worker"},
+            },
+            show_only=[
+                "templates/flower/flower-deployment.yaml",
+                "templates/pgbouncer/pgbouncer-deployment.yaml",
+                "templates/scheduler/scheduler-deployment.yaml",
+                "templates/statsd/statsd-deployment.yaml",
+                "templates/triggerer/triggerer-deployment.yaml",
+                "templates/webserver/webserver-deployment.yaml",
+                "templates/workers/worker-deployment.yaml",
+            ],
+        )
+
+        for doc in docs:
+            component = doc['metadata']['labels']['component']
+            priority = doc['spec']['template']['spec']['priorityClassName']
+
+            if component == "flower":
+                assert priority == "low-priority-flower"
+            elif component == "pgbouncer":
+                assert priority == "low-priority-pgbouncer"
+            elif component == "scheduler":
+                assert priority == "low-priority-scheduler"
+            elif component == "statsd":
+                assert priority == "low-priority-statsd"
+            elif component == "triggerer":
+                assert priority == "low-priority-triggerer"
+            elif component == "webserver":
+                assert priority == "low-priority-webserver"
+            elif component == "worker":
+                assert priority == "low-priority-worker"
+            else:
+                assert False
```

Comment:
Maybe this instead?
```suggestion
            assert priority == f"low-priority-{component}"
```

Reply:
Thanks, thought about this, but didnt realize it also covers the 'assert False condition.